netsetup/__main__.py

```diff
@@ -81,28 +81,28 @@ def main():
     with context:
         dbus.mainloop.glib.DBusGMainLoop(set_as_default=True)
 
-        wpan = Wpantun()
-        bluetooth = Ble(wpan, args.ad_name)
-        dbus_obj = DesktopBus(wpan)
+        wpan_obj = Wpantun()
```

Comment:
I think `instance` or `inst` fits better than `object/obj`. What do you think?

Reply:
I agree